src/main/java/hudson/plugins/ansicolor/action/ColorizedAction.java

```diff
@@ -87,7 +95,20 @@ public static ColorizedAction parseAction(MarkupText text, Run<?, ?> run) {
             final String id = line.substring(from, to);
             return run.getActions(ColorizedAction.class).stream().filter(a -> id.equals(a.getId())).findAny().orElse(CONTINUE);
         }
-        return line.contains(TAG_PIPELINE_INTERNAL) ? IGNORE : CONTINUE;
+        if (line.contains(TAG_PIPELINE_INTERNAL)) {
+            return IGNORE;
+        }
+        final boolean isBuilding = run.isBuilding();
+        LOGGER.fine("Run is building: " + isBuilding);
```

Comment:
I've seen `.finest` elsewhere, same as the comment above, make it consistent. Potentially remove logging where it's not useful anymore and was there fore debugging.

Reply:
I used it because only `fine` or more severe messages are show by default in failed test output on `https://ci.jenkins.io/`. I will gladly get rid of this entirely but I think it's worth waiting a bit and allowing the whole project to be built a couple of times more - if the same test starts failing again the output will be there from the beginning.